src/libraries/sendtohelix.proj

```diff
@@ -11,10 +11,10 @@
     
     <!-- We need to enable xunit reporter so that it parses test results
          Package testing doesn't run on xunit. -->
-    <EnableXunitReporter Condition="'$(TargetGroup)' != 'AllConfigurations'">true</EnableXunitReporter>
+    <EnableXunitReporter Condition="'$(PackageTesting)' != 'true'">true</EnableXunitReporter>
```

Comment:
I think what @ericstj and I meant in the standup was to use the already existing property `BuildAllConfigurations` which is set here: https://github.com/dotnet/runtime/blob/6eb3edf818ced06d698c4148520a86a54c5f427c/eng/build.ps1#L143. Can't you pass down and use that instead?

Reply:
Sure, that is only set when using build.ps1 though, so I'll have to pass it down myself, but sure. (PackageTesting, was more clear to me from the `sendtohelix` perspective, but anyhow.

Reply:
Done.

Reply:
Yes that's because we unfortunately don't use the root build.cmd/sh script with the -projects option. Otherwise we would be able to use the exact same switches. Example: https://github.com/dotnet/runtime/blob/master/src/coreclr/build.cmd#L878-L879. just an FYI

Reply:
I tried, but somehow going through Build.proj messed up the evaluation of the configuration system for sendtohelix.proj and TargetsWindows property was not set, and this has been longer than I would like to, I don't think is worth spending more time on that since I guess this can be done after @Anipik's refactor of the config system.